src/react-apps/applications/runtime/src/App.tsx

```diff
@@ -11,23 +11,57 @@ import LanguageActions from './shared/resources/language/languageActions';
 import PartyActions from './shared/resources/party/partyActions';
 import ProfileActions from './shared/resources/profile/profileActions';
 import TextResourcesActions from './shared/resources/textResources/actions';
-
+import { get } from './utils/networking';
 import {
   languageUrl,
   profileApiUrl,
+  refreshJwtTokenUrl,
 } from './utils/urlHelper';
 
 const theme = createMuiTheme(AltinnAppTheme);
 
+const ONE_MINUTE_IN_MILLISECONDS: number = 60000;
+
 export default function() {
+  let lastRefreshTokenTimestamp: number = 0;
+
+  function setUpEventListeners() {
+    window.addEventListener('mousemove', refreshJwtToken);
+    window.addEventListener('scroll', refreshJwtToken);
+    window.addEventListener('onfocus', refreshJwtToken);
+  }
+
+  function removeEventListeners() {
+    window.removeEventListener('mousemove', refreshJwtToken);
+    window.removeEventListener('scroll', refreshJwtToken);
+    window.removeEventListener('onfocus', refreshJwtToken);
+  }
+
+  function refreshJwtToken() {
+    const timeNow = Date.now();
+    if ((timeNow - lastRefreshTokenTimestamp) > ONE_MINUTE_IN_MILLISECONDS) {
+      lastRefreshTokenTimestamp = timeNow;
+      get(refreshJwtTokenUrl)
+      .then(() => {
+        console.log('token refreshed');
+      })
+      .catch((err) => {
+        console.error('token not refreshed', err);
```

Comment:
if unauthorized (token expired) redirect to authentication component for new round of authentication